src/notebook/index.css

```diff
@@ -9,6 +9,11 @@
 }
 
 
+.p-Widget * + p { 
```

Comment:
I think we should remove this rule from this PR and open a separate issue about it with a full explanation. Then, we can design the document structure and class names (if needed) to make it easy to select and style. The rest of this PR seems reasonable.